CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Developing hcloud-cli
+
+## Generated files
+This repository contains generated files, mainly for testing purposes. These files are generated by running
+`go generate ./...` in the root directory of this repository. Make sure to keep generated files up-to-date
+when making changes to the code.
+
+
+## Unit tests
+Unit tests are located in the `internal` directory. Run them with `go test ./...`.
```

Comment:
With a full code block the command maybe copied, I think it's worth using them:

```sh
go test ./...
```

Reply:
Good idea! In JetBrains IDEs it also lets you run the code snippet directly from the Markdown file.
Implemented in 12cba30